tasks/connectors/edgescan/edgescan.rb

```diff
@@ -71,24 +70,27 @@ def self.metadata
       end
 
       def run(opts)
+        if opts[:option] == "help"
+          print_task_help self.class.metadata[:id]
+          print_good "Returning!"
```

Comment:
Is this left over from debugging?

Reply:
I added that in as it seems that kenna connector does not properly display the options of a task.

For example if i did `docker run toolkit task=edgescan help` it outputs the same as if I did `docker run help`. It does not display edgescan's options.